Tests/HACache.test.swift

```diff
@@ -673,6 +673,15 @@ internal class HACacheTests: XCTestCase {
 
         XCTAssertEqual(populateCount, 1)
     }
+
+    func testInitSubscribingWithoutPopulate() {
+        let cache = HACache(connection: connection, subscribe: subscribeInfo)
+        let result = cache.subscribe { _, _ in }
+        XCTAssertEqual(cache.subscribeInfo?.count, 1)
+        XCTAssertNil(cache.populateInfo)
+        XCTAssertNotNil(cache.connection)
+        XCTAssertNotNil(result)
+    }
```

Comment:
this isn't really testing much, can you write test cases that invoke the code paths?